src/bubbleSort.js

```diff
@@ -0,0 +1,16 @@
+export default function bubbleSort(array){
+
```

Comment:
Apply to entire PR: use `let` when you plan to reassign the variable you're declaring, use `const` when you don't.